app/graphql/mutations/create_ont_libraries_mutation.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Mutations
+  # Mutation to handle the update of a Well.
+  class CreateOntLibrariesMutation < BaseMutation
+    argument :arguments, Types::Inputs::Ont::LibrariesInputType, required: true
+
+    field :tubes, [Types::Outputs::TubeType], null: false
```

Comment:
Not sure but I think the pattern for errors is that the object comes back `nil` rather than an empty array.

Reply:
Change made to instead return nil